### ISSUE TITLE
Harmony freezes on Linux

### ISSUE THREAD

**Issue by brpaz** [vincelwt/harmony#106]
Hello.
I have installed harmony and added my Spotify and Google Music accounts as well as configured my local library.
Now everytime I open harmony it says "loading" on a green bar in the left menu  but I cant click anywhere. sometime later it will just display a blank screen.

I have enabled logging and here is the result:

I have tons of errors like this:

> 
> [15716:1112/110100:INFO:CONSOLE(106)] "Metadatas not found :undefined", source: /opt/Harmony/resources/app/app/js/plugins/local.js (106)
> [15716:1112/110100:INFO:CONSOLE(106)] "Metadatas not found :undefined", source: /opt/Harmony/resources/app/app/js/plugins/local.js (106)
> [15716:1112/110100:INFO:CONSOLE(106)] "Metadatas not found :undefined", source: /opt/Harmony/resources/app/app/js/plugins/local.js (106)
> [15716:1112/110100:INFO:CONSOLE(106)] "Metadatas not found :undefined", source: /opt/Harmony/resources/app/app/js/plugins/local.js (106)

Also, in the begiging I also got some errors saying "Too many open files".

Any ideas?

Thank you.

**Comment by vincelwt** [comment]
Hi,
Could you try without adding a local folder see if it works ?
If you can't access the settings, try resetting the app by removing the `~/.config/configstore/harmony.json` file

**Comment by brpaz** [comment]
Ok. I removed my local files and it seems to work fine.

**Comment by vincelwt** [comment]
Ok so the problem must come from your local files. How much local musics did you tried to add ? More than 30gb ?

**Comment by brpaz** [comment]
Nope. I just added "My music" which has around 6 GB.

**Comment by vincelwt** [comment]
Are all the tracks mp3s ?

**Comment by brpaz** [comment]
well, I noticed I have some .wav files and also some files that are not music files like some images.

**Comment by vincelwt** [comment]
This shouldn't be a problem :/ 
Could you try copying a folder of music files (an album for example) to another directory and add it in the app see if it works ?

**Comment by ArminNaish** [comment]
Same problem here.
I am Harmony on linux using local folder ~/Music loading 16 Gig of music. 
The folder contains everything mp3's, wav-files and images...
The app tries to load the music for 5 minutes. After that I can see "Error - Retry" Message.
After the error appears, the app freezes.

Is there some kind of log-file I can post?

**Comment by vincelwt** [comment]
Hey,
Is this still happening on 0.5 ? I made a bunch of changes to the local files plugin.

**Comment by holms** [comment]
You have best ever player on linux but it's just freezes when trying to pointing to a music folder from external drive. I have to kill a process, and once restarted it's asking for plugin selection again, although in plugin tabs everything is selected. Happens on Debian stretch and the only way to launch app again is to destroy settings, or I'll stuck forever with that plugin window.. So it's not usable at all. I've tried to load just a single dir, which worker, but couldn't figure out how to sort songs. I have DWM in here, so your app it maximized to a full screen, on sort button found :)

**Comment by vincelwt** [comment]
@holms I don't think this is the same issue as the thread. How big is your music folder?

**Comment by holms** [comment]
@vikene sorry for this maybe it's worth to start a new one, would be so much to describe in there. It's rather big around - 100gb. You need some kind of side process to scan music folder. I'd even use some side software to create such library if possible.

**Comment by vincelwt** [comment]
@holms Yeah there are some problems with large libraries. It's probably this issue: https://github.com/vincelwt/harmony/issues/134 right?

**Comment by holms** [comment]
Yeap it's this one.
